app/src/main/java/fr/free/nrw/commons/category/CategoryClient.java

```diff
@@ -30,14 +30,15 @@ public CategoryClient(CategoryInterface CategoryInterface) {
 
     /**
      * Searches for categories containing the specified string.
+     * It will be converted to Lower Case as the server performs Case-insensitive search.
      *
      * @param filter    The string to be searched
      * @param itemLimit How many results are returned
      * @param offset    Starts returning items from the nth result. If offset is 9, the response starts with the 9th item of the search result
      * @return
      */
     public Observable<String> searchCategories(String filter, int itemLimit, int offset) {
-        return responseToCategoryName(CategoryInterface.searchCategories(filter, itemLimit, offset));
+        return responseToCategoryName(CategoryInterface.searchCategories(filter.toLowerCase(), itemLimit, offset));
```

Comment:
Don’t change CategoryClient directly. Change the top layer method that is passing the filter value. In this case it would be the corresponding fragment.